lib/assets/javascripts/jquery.mentionsInput.js

```diff
@@ -7,8 +7,8 @@
  * Using underscore.js
  *
  * License: MIT License - http://www.opensource.org/licenses/mit-license.php
- * 
- * Modifcations for Diaspora:
+ *
+ * Modifications for Diaspora:
```

Comment:
Read the lines coming after this. They detail the changes made to this file by us that are not upstream. Please add your change to it.